mingw-w64-libminpack/PKGBUILD

```
@@ -0,0 +1,82 @@
+# Maintainer: luau-project <luau.project@gmail.com>
+
+_realname=libminpack
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=19961126+dfsg1.5
+_pkgver=19961126+dfsg1-5
+pkgrel=1
+pkgdesc="A software for solving nonlinear equations and nonlinear least squares problems (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='https://salsa.debian.org/science-team/minpack'
```

```suggestion
url='https://salsa.debian.org/science-team/minpack'
```
again no debian!
use the official website.